packages/zosfiles/__tests__/__system__/api/methods/create/Create.system.test.ts

```diff
@@ -106,6 +106,76 @@ describe("Create data set", () => {
     }, LONGER_TIMEOUT);
 });
 
+describe("Allocate Like", () => {
+    let dsnameLike: string;
+    const options: ICreateDataSetOptions = {
+        dsorg: "PO",
+        alcunit: "CYL",
+        primary: 20,
+        recfm: "FB",
+        blksize: 6160,
+        lrecl: 80,
+        showAttributes: true
+    } as any;
+
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({
+            testName: "zos_create_dataset_like"
+        });
+        defaultSystem = testEnvironment.systemTestProperties;
+
+        REAL_SESSION = TestEnvironment.createZosmfSession(testEnvironment);
+        dsnameLike = `${dsname}.LIKE`;
+
+        await Create.dataSet(REAL_SESSION, CreateDataSetTypeEnum.DATA_SET_CLASSIC, dsname, options);
+    });
+
+    afterAll(async () => {
+        await TestEnvironment.cleanUp(testEnvironment);
+    });
+
+    beforeEach(async () => {
+        try {
+            await Create.dataSet(REAL_SESSION, CreateDataSetTypeEnum.DATA_SET_CLASSIC, dsname, options);
+        } catch (error) {
+            Imperative.console.info("Error: " + inspect(error));
+        }
+    });
+
+    afterEach(async () => {
+        try {
+            await Delete.dataSet(REAL_SESSION, dsname);
+            await Delete.dataSet(REAL_SESSION, dsnameLike);
+        } catch (error) {
+            Imperative.console.info("Error: " + inspect(error));
+        }
+    });
+
+    it("should be able to allocate like from a sequential data set", async () => {
+        const options2: ICreateDataSetLikeOptions = {
+            like: dsname,
+            showAttributes: true
+        } as any;
+        let error;
+        let response;
+
+        try {
+            response = await Create.dataSetLike(REAL_SESSION, dsnameLike, options2);
+            Imperative.console.info("Response: " + inspect(response));
+        } catch (err) {
+            error = err;
+            Imperative.console.info("Error: " + inspect(error));
+        }
+
+        expect(error).toBeFalsy();
+        expect(response).toBeTruthy();
+
+        expect(response.success).toBe(true);
+        expect(response.commandResponse).toContain(ZosFilesMessages.dataSetCreatedSuccessfully.message);
+        expect(response.commandResponse).toContain(options);
```

Comment:
This `expect` fails, because `TextUtils.prettyJson` generates YAML rather than JSON output, and the attributes returned by `List.dataSet` aren't exactly identical to the options provided to `Create.dataSet`.